--- origsrc/libopensync-plugin-kdepim-0.22/src/Makefile.am	2007-03-27 06:49:57.000000000 -0500
+++ src/libopensync-plugin-kdepim-0.22/src/Makefile.am	2008-05-09 06:21:06.078125000 -0500
@@ -32,6 +32,7 @@
 # ugly hack to make the c++ library work
 kdepim_sync_la_LIBADD = -lstdc++ @OSYNC_LIBS@
 
+BUILT_SOURCES = KNotesIface_stub.cpp KNotesIface_stub.h
 $(srcdir)/kdepim_impl.cpp: KNotesIface_stub.h
 KNotesIface_stub.cpp: KNotesIface_stub.h
 
